README.md

```diff
@@ -1,2 +1,12 @@
 # drb-network-prep
-Code repo to prepare the river network for modeling in the Delaware River Basin
+Code repo to prepare the river network for modeling in the Delaware River Basin  
+  
+## Prep the DRB river network for modeling  
+Nothing here yet. Functions to prep the DRB network are currently included in the delaware-model-prep [repo](https://github.com/USGS-R/delaware-model-prep).  
```

Comment:
Are you planning to add more to this repo?

Reply:
Possibly. The delaware-model-prep repo is pretty bulky, so we've talked about moving the network prep steps outside of that repo if/when we shift from GFv1 to GFv2.